packages/driver/src/cy/commands/fixtures.ts

```diff
@@ -12,15 +14,73 @@ const clone = (obj) => {
   return JSON.parse(JSON.stringify(obj))
 }
 
+/**
+ * Given a path, returns an array containing the path with and without its extension.
+ * If there is no extension, returns an array containing only the original path.
+ *
+ * Used so invalidation can match both "foo.json" and "foo".
+ *
+ * @returns [pathWithExtension, pathWithoutExtension] if extension exists, otherwise [path].
+ */
+const withAndWithoutExt = (path: string) => {
+  const extension = extname(path)
+
+  return extension ? [path, path.slice(0, -extension.length)] : [path]
+}
+
+/**
+ * Builds path prefixes that might have been used in a cache key's fixture
+ * portion. Includes forward and backslash variants, with and without
+ * extensions, and lowercase variants on Windows.
+ */
+const buildPrefixes = (rawPath: string): string[] => {
+  const forward = rawPath.split(sep).join('/')
+  const backslash = forward.replace(/\//g, '\\')
+
+  const bases = [
+    ...withAndWithoutExt(forward),
+    ...withAndWithoutExt(backslash),
+  ]
+
+  if (process?.platform === 'win32') {
```

Comment:
The polyfill actually doesn't include the platform in the browser so this never gets called. By the Cypress global does provide it
```suggestion
  if (Cypress.platform === 'win32') {
```